packages/fx-core/templates/plugins/resource/hosting/bicep/webapp/config.template.bicep

```diff
@@ -1,5 +1,5 @@
 // Get existing app settings for merge
-var botCurrentAppSettings = list('${provisionOutputs.webAppOutput.value.webAppResourceId}/config/appsettings', '2021-02-01').properties
+var botCurrentAppSettings = list('${provisionOutputs.webAppOutput.value.botWebAppResourceId}/config/appsettings', '2021-02-01').properties
```

Comment:
Will this change cause backward compatibility issue to handle projects created by Toolkit GA version?

Reply:
No. It uesd to be `botWebAppResourceId`.